implementation/src/main/java/io/smallrye/mutiny/groups/UniOnNotNull.java

```diff
@@ -198,4 +198,29 @@ public UniOnNotNull(Uni<T> upstream) {
         });
     }
 
+    /**
+     * If the current {@link Uni} emits an item, the produced {@link Uni} emits the retrieved failure.
```

Comment:
```suggestion
     * If the current {@link Uni} emits an item, the produced {@link Uni} emits the retrieved failure
```

Or the sentence is cut.